src/onegov/election_day/formats/vote/wabstic.py

```diff
@@ -149,7 +128,8 @@ def import_vote_wabstic(vote, principal, number, district,
 
         # Check if the entity is counted
         try:
-            counted = False if int(line.sperrung or 0) == 0 else True
+            counted_num = validate_integer(line, 'sperrung')
+            counted = False if counted_num == 0 else True
```

Comment:
Same as above. Is `counted_num` maybe not a good name? It seems like you want to say that something is counted, if the number of uncounted values is 0.

Reply:
counted here refers hour of locking of the result, so it would be better to make that clear using meaningful variables.